packages/fether-electron/src/main/app/methods/setupRequestListeners.js

```diff
@@ -6,8 +6,36 @@
 import electron from 'electron';
 
 import messages from '../messages';
+import { TRUSTED_URLS } from '../constants';
+import Pino from '../utils/pino';
+
+const pino = Pino();
 const { ipcMain, session } = electron;
 
+pino.debug(
+  'Configuring Content-Security-Policy for environment: ',
+  process.env.NODE_ENV
+);
+// Note: `worker-src blob:` required for the webcam
+const CSP =
+  process.env.NODE_ENV === 'development'
+    ? `
+    default-src 'self';
+    script-src 'self' file: http: blob: 'unsafe-inline' 'unsafe-eval';
+    connect-src 'self' file: http: https: ws: wss:;
+    img-src 'self' 'unsafe-inline' file: data: blob: http: https:;
+    style-src 'self' 'unsafe-inline' file: blob:;
+    worker-src blob:;
```

Comment:
An idea: Could you define these CSPs as an object/array, so that we can put comments to explain them?

Reply:
@amaurymartiny I've pushed a commit that defines the CSP as an object and avoids duplication